src/util/articleListMainPage.ts

```diff
@@ -6,7 +6,7 @@ export function makeArticleListItem(dump: Dump, articleEntry: any) {
 }
 
 export function makeArticleImageTile(dump: Dump, articleEntry: ArticleDetail, webp: boolean) {
-    return `<a class="item" href="${'./' + encodeArticleIdForZimHtmlUrl(articleEntry.title.replace(/ /g, '_') || '')}">
+    return `<a class="item" href="${encodeArticleIdForZimHtmlUrl(articleEntry ? './' + articleEntry.title.replace(/ /g, '_') : './')}">
```

Comment:
in case the `articleEntry.title` starts with a `/` it will try to replace it with a `./` and we are already adding a `./` to it after calling `encodeArticleIdForZimHtmlUrl()` one thing that I did wrong is use `articleEntry?` in place of `articleEntry.title` sorry about that.....but other than that even if `articleentry.title`was NULL we are actually having `href` as `./` that's why I kept `./` even in case `articleEntry.title` is null